subsys/nrf_rpc/utils/client/utils_client.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "zephyr/sys/util.h"
+#include <nrf_rpc/nrf_rpc_serialize.h>
+#include <nrf_rpc_cbor.h>
+
+#include <ncs_commit.h>
+#include <utils_rpc_ids.h>
+
+NRF_RPC_GROUP_DECLARE(utils_group);
+
+static char version[256];
+
+const char *get_version()
+{
+	struct nrf_rpc_cbor_ctx ctx;
+	size_t size = ARRAY_SIZE(version);
+	const void *buf = NULL;
+
+	memset(version, 0, ARRAY_SIZE(version));
+
+	NRF_RPC_CBOR_ALLOC(&utils_group, ctx, 0);
+
+	nrf_rpc_cbor_cmd_rsp_no_err(&utils_group, UTILS_RPC_GET_VERSION, &ctx);
+
+	buf = nrf_rpc_decode_str(&ctx, version, size);
```

Comment:
The function has already copied the string and added eos in version buffer. One more copying is not required.